google-cloud-spanner/src/main/java/com/google/cloud/spanner/SessionPoolOptions.java

```diff
@@ -334,8 +334,7 @@ static InactiveTransactionRemovalOptions.Builder newBuilder() {
     }
 
     static class Builder {
-      private ActionOnInactiveTransaction actionOnInactiveTransaction =
-          ActionOnInactiveTransaction.WARN;
+      private ActionOnInactiveTransaction actionOnInactiveTransaction;
```

Comment:
Just to be sure: Are we sure that all code can safely handle that this value is now `null`?

Reply:
Yes, all places safely handle the null value.